common/utils.go

```diff
@@ -6,11 +6,10 @@ import (
 	"math/rand"
 	"time"
 
-	"github.com/dgrijalva/jwt-go"
```

Comment:
Sure? Might cause error can'f find `jwt`? I think we should delete all package in the vendor or GOPATH, and make sure all packages can be installed in a new environment.

Reply:
OK, I make sure it later on.

Reply:
In fact, travis CI should do the work... but the gothinkster's repo didn't get right config... Let's wait Eric reply..

Reply:
OK